base_swivel:
    ID: 1

vertical_tilt:
    ID: 2

#Extension should be in multi-turn mode        
arm_extension:
    ID: 3
    Return_Delay_Time: 0
    CW_Angle_Limit: 0
    CCW_Angle_Limit: 2048
        
wrist:
    ID: 4

wrist_tilt:
  ID: 5

gripper:
  ID: 6